wifiphisher/pywifiphisher.py

```diff
@@ -137,6 +149,12 @@ def check_args(args):
         sys.exit(
             '[' + R + '-' + W + '] Lure10 capture does not exist. Listing directory of captures: ' + str(os.listdir(LOCS_DIR)))
 
+    if (args.mac_ap_interface and args.no_mac_randomization) or \
+            (args.mac_deauth_interface and args.no_mac_randomization):
+        sys.exit(
+            '[' + R + '-' + W + '] --mac-ap-interface (-iAM) or (--mac-deauth-interface (-iDM)) '
+            'and --no-mac-randomization (-iNM) cannot work together')
```

Comment:
I would rephrase this to "--no-mac-randomization (-iNM) cannot work together with --mac-ap-interface (-iAM) or --mac-deauth-interface (-iDM)"